extensions/sql-database-projects/package.json

```diff
@@ -330,31 +303,31 @@
       ],
       "objectExplorer/item/context": [
         {
-          "command": "sqlDatabaseProjects.importDatabase",
+          "command": "sqlDatabaseProjects.createProjectFromDatabase",
           "when": "connectionProvider == MSSQL && nodeType && nodeType == Database && mssql:engineedition != 11",
           "group": "export"
         },
         {
-          "command": "sqlDatabaseProjects.importDatabase",
+          "command": "sqlDatabaseProjects.createProjectFromDatabase",
```

Comment:
the command contribution can be simplified to just one item per host if you use regex like I did, nodeType equals database or server type